modules/images-configuration-parameters.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-parameters_{context}"]
+= Image controller configuration parameters
+
+The `image.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`metadata`
+|Standard object's metadata.
+
+|`spec`
+|Holds user-settable values for configuration.
+
+|`status`
+|Holds observed values from the cluster.
```

Comment:
unlike what i said for the build object, maybe there is value in making it clear that the spec subsection is for user editing and the status subsection is for observed settings.  But they are subsections, not fields themselves, effectively.